module-system/module-implementations/sov-evm/src/genesis.rs

```diff
@@ -45,6 +46,11 @@ impl<C: sov_modules_api::Context> Evm<C> {
             chain_id: config.chain_id,
             limit_contract_code_size: config.limit_contract_code_size,
             spec,
+            coinbase: config.coinbase,
+            block_gas_limit: config.block_gas_limit,
+            starting_base_fee: U256::from_limbs([config.starting_base_fee, 0u64, 0u64, 0u64])
```

Comment:
Both `EvmChainCfg` and `EvmConfig` so we can change type of ` starting_base_fee` in one of the and avoid the conversion.

Reply:
starting_base_fee will be used in genesis, removed from `EvmChainCfg`